cli/cmd/install.go

```diff
@@ -48,8 +49,21 @@ var installCmd = &cobra.Command{
 		ns := cmd.Flag("namespace").Value.String()
 		cmd.Flags().StringSliceVar(&ignoredNamespaces, "ignore-namespace", DefaultIgnoredNamespaces, "--ignore-namespace foo logging")
 		fmt.Printf("Installing Odigos version %s in namespace %s ...\n", versionFlag, ns)
+
+		isOdigosCloud := odigosCloudApiKeyFlag != ""
 		createKubeResourceWithLogging(ctx, fmt.Sprintf("Creating namespace %s", ns),
 			client, cmd, ns, createNamespace)
+		if isOdigosCloud {
+			createKubeResourceWithLogging(ctx, "Creating Odigos Cloud Secret",
+				client, cmd, ns, createOdigosCloudSecret)
+			createKubeResourceWithLogging(ctx, "Creating Own Telemetry Pipeline",
+				client, cmd, ns, createOwnTelemetryPipeline)
+			createKubeResourceWithLogging(ctx, "Deploying Odigos Cloud Proxy",
+				client, cmd, ns, createKeyvalProxy)
+		} else {
+			createKubeResourceWithLogging(ctx, "Own Telemetry Pipeline disabled",
```

Comment:
Not sure we should print it to the screen, may sounds to user like there is some kind of error. Maybe just call the function without logging this text

Reply:
good point. fixed 👍